cgnsutilities/cgnsutilities.py

```diff
@@ -3019,7 +3019,7 @@ def convertPlot3d(plot3dFile, cgnsFile):
     libcgns_utils.utils.convertplot3d(plot3dFile, cgnsFile)
 
 
-def mirrorGrid(grid, axis, tol):
+def mirrorGrid(grid, axis, tol, actualName=False):
```

Comment:
You should add this option to the CLI as well. I also think the name of the kwarg could be more descriptive. What do you think @marcomangano?

Reply:
Yeah agreed, haven't used this in a while but it should be better to have it in the CLI (and maybe default to `True`?). As for the kwarg, something along the lines of `keepNames` could be more intuitive?

Reply:
Agree, I changed it. I think having an option seems more familiar than kwarg.